Traceback (most recent call last):
  File "examples/http3_server.py", line 431, in <module>
    secrets_log_file = open(args.secrets_log, "a")
FileNotFoundError: [Errno 2] No such file or directory: '/results/temp/aioquic_key.log"'
